samples/nrf9160/mqtt_simple/overlay-tls.conf

```diff
@@ -6,7 +6,8 @@
 
 # MQTT TLS
 CONFIG_MQTT_LIB_TLS=y
-CONFIG_MQTT_BROKER_PORT=8883
+CONFIG_MQTT_BROKER_PORT=8886
+CONFIG_MQTT_TLS_PEER_VERIFY=1
```

Comment:
I am using a different broker, but one that also verifies with the same Let's Encrypt CA.  This broker has an issue with peer verification, which I learned about in this MQTT with TLS walkthrough on our devzone. I will try test.mosquitto.org with one of its other TLS ports which require a different CA.

https://devzone.nordicsemi.com/guides/cellular-iot-guides/b/software-and-protocols/posts/enabling-and-testing-tls-in-mqtt_5f00_simple

> NOTE: PEER_VERIFY must not be set to VERIFY_REQUIRED (2) because test.mosquitto.org does not verify correctly, probably because the certificates it uses are not strong enough (SHA1 and 1024-bit RSA).